src/main/webapp/app/lti/lti13-exercise-launch.component.ts

```diff
@@ -57,9 +64,30 @@ export class Lti13ExerciseLaunchComponent implements OnInit {
                         console.error('No LTI targetLinkUri received for a successful launch');
                     }
                 },
-                error: () => {
```

Comment:
This is quite lengthy. Could you separate the next and error handling in methods to make this a little bit easier to understand?

Reply:
I've separated the next and error handling into distinct methods, hope it is more understandable now.